README.md

```diff
@@ -500,6 +503,75 @@ status:
     status: 'True'
 ```
 
+## Custom Binding Paths
+
+There are scenarios where the application resource is not PodSpec-able and yet bind `volume`, `volumeMount`, and `env` at custom locations.  This extension provides the customized binding functionality through a `BindingPathMap` resource.  The `ServiceBinding` reconciler **MUST** watch for the `BindingPathMap` and the `ServiceBinding` resources and react.  The `BindingPathMap` resource **MUST** be a cluster-scoped resource.  The allowed custom binding paths are `volume`, `volumeMount`, and `env`.  The value for each of these custom binding paths **MUST** be a string type.  If `volumeMount` is specified, `volume` also **MUST** be defined.  The values of these items **MUST** be dot-delimited paths pointing to the fields on the application resource.  The reconciler **MUST** bind each of these specified values in application resource at the given location.  An exemplar CRD can be found [here][cbp-crd].
+
+1. The `volume` field in the application resource **MUST** conform to the [Volume][volume] API.
+2. The `volumeMount` field in the application resource **MUST** conform to the [VolumeMount][volumemount] API.
+3. The `env` field in the application resource **MUST** conform to the [EnvVar][env] API.
+
+A `BindingPathMap` resource **MUST** define a `.status.conditions` which is an array of `Condition` objects.  A `Condition` object **MUST** define `type`, `status`, and `lastTransitionTime` entries.  At least one condition containing a `type` of `Ready` **MUST** be defined.  The `status` of the `Ready` condition **MUST** have a value of `True`, `False`, or `Unknown`.  The `lastTransitionTime` **MUST** contain the last time that the condition transitioned from one status to another.  A `BindingPathMap` resource **MAY** define `reason` and `message` entries to describe the last `status` transition.  There could be multiple application resources matching the `apiVersion` and `kind`.  Partial failures **MUST** be aggregated and reported on the binding status's `Ready` condition.
```

Comment:
> Partial failures **MUST** be aggregated and reported on the binding status's `Ready` condition.

What are the possible failure modes? Either the referenced gvk exists or it doesn't.